src/modules/fancyzones/dll/dllmain.cpp

```diff
@@ -197,6 +219,8 @@ class FancyZonesModule : public PowertoyModuleIface
     std::vector<HWINEVENTHOOK> m_staticWinEventHooks;
     HWINEVENTHOOK m_objectLocationWinEventHook;
 
+    bool disableCore{ false };

```

Comment:
Let's use a better name for this variable.

Reply:
Best name I came up with.
```
// If non-recoverable error occurs during initialization of FancyZones module, disable
// its core functionality. Accessing FancyZones settings will still be possible.
bool disableFancyZonesModuleCore{ false };
```